impl/src/bitfield.rs

```diff
@@ -116,7 +110,7 @@ impl BitfieldStruct {
                 /// The returned byte slice is layed out in the same way as described
                 /// [here](https://docs.rs/modular-bitfield/#generated-structure).
                 #[inline]
-                pub fn to_bytes(&self) -> &[u8] {
+                fn to_bytes(&self) -> &[u8] {
```

Comment:
We should somehow regulate this because some people might depend on this method for a purpose. After all it is a feature that we can turn our bitfields into byte slices. Hiding this feature from users would make this crate less valueable.

Reply:
Another idea might be to make this public if all fields are public.

Reply:
And yet another idea is to allow users to configure it via some attributes:
```rust
#[bitfield(to_bytes)]
```